teamB/eunsong/routes/index.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router();
+
+const userRouter = require("./userRouter");
+const postRouter = require("./postRouter");
+const likeRouter = require("./likeRouter");
+router.use("/users", userRouter.router);
+router.use("/posts", postRouter.router);
+router.use("/likes", likeRouter.router);
```

Comment:
- 해당 부분은 코드 가독성을 위해 한 줄 띄어서 작성해 주세요 !
```suggestion
const likeRouter = require("./likeRouter");

router.use("/users", userRouter.router);
router.use("/posts", postRouter.router);
router.use("/likes", likeRouter.router);
```

Reply:
네 한줄 띄어서 다시 push 하겠습니다!